hdx_hapi/db/dao/util/util.py

```diff
@@ -92,6 +92,20 @@ def apply_location_admin_filter(
     return query
 
 
+class EntityWithLocationAdmin(Protocol):
```

Comment:
I think this part of the code got "doubled" somehow (appears twice)

Reply:
Yes - the linter picked this up